ONSdigital/response-operations-ui
# Post case event to transition casegroup status

# Motivation and Context
Previously we were transitioning casegroup status with a call to the `/casegroups/transitions/...` endpoint. This doesn't align with other case lifecycle processes.
We now post a case event to trigger this transition 

# What has changed
Removed code to post to `/casegroups/transitions/...` endpoint and replaced it with a post to case event endpoint.

# How to test?
Check that changing casegroup status in the UI updates the required rows in the case database, 
i.e. casegroup status is updated and the related case is set to `INACTIONABLE`

# Links
[Trello](https://trello.com/c/xkWvMZFk/420-use-case-event-rather-than-case-group-transition-for-change-response-status)

## Associated PR's
[rm-case-service](https://github.com/ONSdigital/rm-case-service/pull/93)
[rm-acceptance-tests](https://github.com/ONSdigital/rasrm-acceptance-tests/pull/196)
